app/javascript/components/pages/SearchResults.js

```diff
@@ -1,43 +1,54 @@
-import React from "react"
-import PropTypes from "prop-types"
-import { Link } from "react-router-dom"
+import React from "react";
+import PropTypes from "prop-types";
+import { Link } from "react-router-dom";
 class SearchResults extends React.Component {
-    render() {
-        return (
-            <React.Fragment>
-                <h3>Your top TICities are...</h3>
-                <div id="search-results-body">
-                    {this.props.searches.map((search, index) => {
-                        return (
-                            <>
-                                <div key={index} className="search-result-card">
-                                    <h3>{search.city1}</h3>
-                                    <br />
-                                    <Link to={`/cityshow/${search.city1}`} city1={search.city1} className="button">
-                                        More Info
-                                    </Link>
-                                </div>
-                                <div key={index} className="search-result-card">
-                                    <h3>{search.city2}</h3>
-                                    <br />
-                                    <Link to={`/cityshow/${search.id}`} className="button">
-                                        More Info
-                                    </Link>
-                                </div>
-                                <div key={index} className="search-result-card">
-                                    <h3>{search.city3}</h3>
-                                    <br />
-                                    {/* <Link to={`/searchresults/${search.id}`} className="button">
-                                    More Info
-                                    </Link> */}
-                                </div>
-                            </>
-                        )
-                    })}
+  render() {
+    return (
+      <React.Fragment>
+        <h3>Your top TICities are...</h3>
+        <div className="body-container">
+          {this.props.searches.map((search, index) => {
+            return (
+              <>
+                <div key={index} className="search-result-card">
+                  <h3>{search.city1}</h3>
+                  <br />
+                  <Link
+                    to={`/cityshow/${search.city1}`}
+                    city1={search.city1}
```

Comment:
Does the `city1` property do anything here?

Reply:
it will once we are no longer using mockdata/scaffolding, we got ahead thinking we were going to be implementing the backend sooner, city1 removed